PWGCF/EbyEFluctuations/Tasks/MeanPtFlucIdentified.cxx

```diff
@@ -628,6 +738,28 @@ struct meanPtFlucId {
     double pt_Pr = 0, Q1_Pr = 0, Q2_Pr = 0, Q3_Pr = 0, Q4_Pr = 0;
     double pt_Ka = 0, Q1_Ka = 0, Q2_Ka = 0, Q3_Ka = 0, Q4_Ka = 0;
 
+    TLorentzVector p1, p2, p;
+    double invMassGamma = 0.0;
+
+    for (auto const& [trkEl, trkPos] : soa::combinations(soa::CombinationsFullIndexPolicy(tracks, tracks))) {
+      if (trkEl.index() == trkPos.index())
+        continue;
+
+      if (!selTrack(trkEl) || !selTrack(trkPos))
+        continue;
+
+      if (!selElectrons(trkEl) || !selElectrons(trkPos))
+        continue;
+
+      p1.SetXYZM(trkEl.px(), trkEl.py(), trkEl.pz(), massEl);
+      p2.SetXYZM(trkPos.px(), trkPos.py(), trkPos.pz(), massEl);
+
+      p = p1 + p2;
+
+      invMassGamma = p.M();
```

Comment:
Consider using `RecoDecay` methods for calculations with momenta and masses.

Reply:
@TGahlaut1 Vit is right, you can use the methods in 
https://github.com/AliceO2Group/O2Physics/blob/master/Common/Core/RecoDecay.h
instead of using `TLorentzVector`. It will be much more efficient